src/utilities.ts

```diff
@@ -0,0 +1,3 @@
+export function clamp (number: number, min: number, max: number) {
```

Comment:
Please move this file to src/game_logic/shared.ts. You can overwrite it because it is unused.